app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -104,15 +103,17 @@ class ConversationViewModel @Inject constructor(
         viewModelScope.launch {
             // TODO what if conversationId is null???
             conversationId?.let { sendTextMessage(it, messageText) }
-
         }
     }
 
     fun sendAttachmentMessage(attachmentBundle: AttachmentBundle?) {
```

Comment:
I know now we are only addressing the image attachments. But I think we can add a `TODO` to this function as a reminder that we will have to add a `type` property on  `AttachmentBundle` to identify the type of attachment, so later here we can call the correct usecase. Let me know what you think =)